src/orca-log-catcher/OrcaLogger.res

```diff
@@ -897,16 +894,14 @@ let make = (
       version: GlobalVars.repoVersion,
       category: USER_EVENT,
       value: "log initiated",
-      internalMetadata: "",
       paymentId: clientSecret.contents->getPaymentId,
       merchantId: merchantId.contents,
       browserName: arrayOfNameAndVersion->Array.get(0)->Option.getOr("Others"),
       browserVersion: arrayOfNameAndVersion->Array.get(1)->Option.getOr("0"),
       platform: Window.Navigator.platform,
       userAgent: Window.Navigator.userAgent,
-      appId: "",
+      appId: Window.Location.hostname,
```

Comment:
this will log the hostname of the iframe
This is not the expected behaviour

Consume hostname from Elements as a postmessage